ert3/config/_ensemble_config.py

```diff
@@ -34,22 +35,55 @@ class ForwardModel(_EnsembleConfig):
     driver: Literal["local", "pbs"] = "local"
 
 
+def _ensure_transformation_for_resources(
```

Comment:
needs to be constraint on transformation such that it only is defined for resources